src/editor/actions/actions.js

```diff
@@ -160,13 +160,49 @@ export function updateSelectedRevisionId(selectedRevisionId) {
 export function getNotebookRevisionList() {
   return (dispatch, getState) => {
     dispatch({ type: "GETTING_NOTEBOOK_REVISION_LIST" });
+    dispatch({
+      type: "UPDATE_NOTEBOOK_HISTORY_BROWSER_HAS_LOCAL_ONLY_CHANGES",
+      hasLocalOnlyChanges: true
+    });
```

Comment:
Hmm, I'm not sure if this clause is necessary? We should figure this out after we've finished downloading the revision list. Also see below for more comments on this reducer. :)